hack/util.sh

```diff
@@ -464,6 +464,26 @@ function util::delete_all_clusters() {
   echo "Deleted all clusters and the log file is in ${log_file}"
 }
 
+# util::delete_necessary_resources deletes clusters(karmada-host, member1, member2 and member3) and related resources directly
+# util::delete_necessary_resourcesactually do three things: delete cluster、remove kubeconfig、record delete log
+# Parameters:
+#  - $1: KUBECONFIG files of clusters, separated by ",", such as "~/.kube/karmada.config,~/.kube/members.config"
+#  - $2: clusters, separated by ",", such as "karmada-host,member1"
+#  - $3: log file path, such as "/tmp/karmada/"
+function util::delete_necessary_resources() {
+  local config_files=${1}
+  local clusters=${2}
+  local log_path=${3}
+
+  local log_file="${log_path}"/delete-necessary-resources.log
+  rm -rf ${log_file}
+  mkdir -p ${log_path}
+
+  rm -f config_file
+  kind delete clusters $clusters >> "${log_file}" 2>&1
```

Comment:
This modification introduces new problem:

* `rm -f config_file`, what does `config_file` refer to? you mean `rm -rf ${config_files}` ?

* the previous tr ',' ' ' needs to be retained, `rm -rf ~/.kube/karmada.config,~/.kube/members.config` is invalid, while `rm -rf ~/.kube/karmada.config ~/.kube/members.config` is valid, you known what I mean?